deployment/quick-start/services-configuration.yaml.template

```diff
@@ -65,6 +65,8 @@ cluster:
 
 
 rest-server:
+#  # launcher type. k8s or yarn
```

Comment:
indent

Reply:
It's following the indent rule of comment's comment.